src/annotator/integrations/html.js

```diff
@@ -0,0 +1,46 @@
+import { anchor, describe } from '../anchoring/html';
+
+import { HTMLMetadata } from './html-metadata';
+
+/**
+ * @typedef {import('../../types/annotator').Integration} Integration
+ */
+
+/**
+ * Document type integration for ordinary web pages.
+ *
+ * This integration is used for web pages and applications that are not handled
+ * by a more specific integration (eg. for PDFs).
+ *
+ * @implements {Integration}
+ */
+export class HTMLIntegration {
+  constructor(container = document.body) {
+    this.container = container;
+    this.anchor = anchor;
+    this.describe = describe;
+
+    this._htmlMeta = new HTMLMetadata();
+  }
+
+  destroy() {
+    // There is nothing to do here yet.
+  }
+
+  contentContainer() {
+    return this.container;
+  }
+
+  fitSideBySide() {
+    // Not yet implemented.
+    return false;
+  }
+
+  async getMetadata() {
+    return this._htmlMeta.getDocumentMetadata();
+  }
```

Comment:
`getMetadata` and `uri` don't need to be async in this implementation, but they are for consistency with `PDFIntegration`.